app/email_utils.py

```diff
@@ -1427,3 +1433,18 @@ def save_email_for_debugging(msg: Message, file_name_prefix=None) -> str:
         return file_name
 
     return ""
+
+
+def get_dmarc_status(msg: Message) -> Optional[DmarcCheckResult]:
```

Comment:
I wonder if we can have some tests for this method to make sure its parsing is correct?

Reply:
:+1:

Reply:
added